### PR TITLE
Reference the in-house Ansible tutorial, from the Ansible-Galaxy tutorial

### PR DESCRIPTION
**Pull request by vladvisan**
Not paramount, but why not mention it since the effort was put into writing it?

### CODE REVIEW

**Comment by vladvisan** [comment]
Maybe also do the mirror version: refer to this tutorial from the Ansible tutorial https://github.com/galaxyproject/training-material/blob/main/topics/admin/tutorials/ansible/tutorial.md?plain=1

I see there is an "Ansible-Galaxy" section there, but no link to this tutorial.

I don't want to spam pull requests, so let me know if this is a good idea.

File: topics/admin/tutorials/ansible-galaxy/tutorial.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ edam_ontology:
 - topic_3071 # Data Management
 ---
 
-This tutorial assumes you have some familiarity with [Ansible](https://www.ansible.com/resources/get-started) and are comfortable with writing and running playbooks. Here we'll see how to install a Galaxy server using an Ansible playbook. The Galaxy Project has decided on Ansible for all of its deployment recipes. For our project, Ansible is even more fitting due to its name:
+This tutorial assumes you have some familiarity with [Ansible](https://www.ansible.com/resources/get-started) and are comfortable with writing and running playbooks.  If not, check out our guide [here](../ansible/tutorial.html) first. Here we'll see how to install a Galaxy server using an Ansible playbook. The Galaxy Project has decided on Ansible for all of its deployment recipes. For our project, Ansible is even more fitting due to its name:
```
[the remark this answers is not in the document]
Comment (vladvisan) [reply topics/admin/tutorials/ansible-galaxy/tutorial.md]: Thank you, I agree, committed.

**Comment by vladvisan** [comment]
Thank you for the feedback.
I'm following the tutorial at the same time and will update the PR at the same time.
I'll post a new comment when it's ready for a new review.

**Comment by hexylena** [comment]
>  I'm following the tutorial and will update the PR at the same time.

when you're done, please add yourself to the `testing` section of the tutorial header (under `contributions`)

**Comment by vladvisan** [comment]
Thank you for the "testing" suggestion.

I have not followed the "install Galaxy with Ansible" tutorial yet, but should be enough progress for now.

1. The "contributors.yaml/organisation.yaml" additions are redundant with my other PR (https://github.com/galaxyproject/training-material/pull/4972) , but will not cause any merge conflict (same change in same location in the files)
2. Feedback on the display of requirements:
   - Thanks, I had not seen it
   - "Hands on: Hands on: " seems like a mistake? Should be 0 "Hands on" since there is a "Hands on" link to the right?
   - I feel like "Galaxy server administration" should not be a hyperlink
   - I feel like "Ansible" should be in bold
   - This seems to be in the website's engine, as opposed to something I can change in the tutorial.md directly
3. There are alot of links that point to version 2.9 of the Ansible doc website, in both tutorials
   - I understand not pointing to /latest/, for stability's sake
   - But I think it would be good to update it to /9/ (Ansible changed version naming conventions, 9==2.17) since 2.9 is 5 years old
   - I tried to it, but it was too time-consuming so I did not have the time
       - It is time-consuming because there are tens of links and about half are invalid if you just replace /2.9/ with /9/
       - I spent some time searching for the corresponding links in the new documentation, but I'm not good enough at Ansible for the moment to do that
4. I think there is a bug in the "details" tag, when the text is longer than one line, causing bugs such as the following:
   - image
   - you can see that the "4. Create the roles directory (...)" is to the right instead of on a new line
   - this is in the "live" version
   - As a patch (not the cleanest solution but it works), I changed the content of the detail-title, to make it smaller than one line, and so the bug does not  happen, and there is a new line

And thanks for writing the tutorial, it helped me understand the basics of Ansible!

**Comment by vladvisan** [comment]
Forgot to mention why I added a lot of "repository (unrelated to the Galaxy Project)", because as a novice, this was very confusing me
Especially since I started by looking at https://github.com/galaxyproject/ansible-galaxy , it took me a while to realise "ansible-galaxy" didn't refer to that, but actually to the CLI tool that comes packaged with Ansible.

File: topics/admin/tutorials/ansible/tutorial.md [hunk 2]
```diff
@@ -295,7 +295,7 @@ The above introduction was certainly not enough for you to feel confident in Ans
 >       > ```
 >       {: .solution }
 >
->       > <details-title>For your own infrastructure, do you connect with a different user? Password? SSH key?</details-title>
+>       > <details-title>For your own infra, do you connect as a different user? Password? SSH key?</details-title>
```
Comment (hexylena) [inline topics/admin/tutorials/ansible/tutorial.md:301]:
```suggestion
>       > <details-title>For your own infrastructure, do you connect as a different user? Password? SSH key?</details-title>
```

we need to keep the longer form of the word here, it should be more accessible for non-native english speakers, as well as those reading the materials through machine translation where infrastructure is more likely to be translated correctly than the informal slang term 'infra' that we all use :)

**Comment by hexylena** [comment]
> "Hands on: Hands on: " seems like a mistake? Should be 0 "Hands on" since there is a "Hands on" link to the right?
> I feel like "Galaxy server administration" should not be a hyperlink
> I feel like "Ansible" should be in bold
> This seems to be in the website's engine, as opposed to something I can change in the tutorial.md directly

yes these seem like reasonable suggestions (the duplicate hands-on is a known issue I hadn't fixed yet.)

We'll take care of them 

> But I think it would be good to update it to /9/ (Ansible changed version naming conventions, 9==2.17) since 2.9 is 5 years old

Yes, it would be good. We'll see about doing that during the next GAT, whenever that is. Outside of the GAT events the materials do not get updated very often because the update process is quite laborious and we want to be sure that all tutorials in the GAT series work together correctly.

You intended this to be a small change, I think that's good :) Please don't feel the need to take on big tasks like that, that can be a task for the team.

I'll file it as a separate issue and add to our GAT roadmap.

> I think there is a bug in the "details" tag, when the text is longer than one line, causing bugs such as the following: 

I will move it into a separate bug, I've suggested reverting that change because it would cause other issues. We will investigate it more thoroughly.

> And thanks for writing the tutorial, it helped me understand the basics of Ansible!

I'm glad it was helpful! I don't think it is a great tutorial, it's not my best work, I think it could be improved a lot but I have not had time for it lately. but good that it is still useful in this state.

**Comment by hexylena** [comment]
Thanks! And welcome to the GTN @vladvisan !

**Comment by vladvisan** [comment]
@hexylena Thank you for the feedback and for creating the related issues.

**Pull request merged**